### PR TITLE
fix: wrap and unwrap nil

### PR DESCRIPTION
**Pull request by ig-ra**

### CODE REVIEW

File: sdk/sdktypes/value_wrap_test.go [hunk 1]
```diff
@@ -66,10 +66,15 @@ func TestValueWrapper(t *testing.T) {
 			w:   sdktypes.NewStringValue("meow"),
 			unw: "meow",
 		},
+		// {
```
Comment (itayd) [inline sdk/sdktypes/value_wrap_test.go]: kill

**Review by itayd — APPROVED** [review]

**Review by itayd — APPROVED** [review]
oh, please add a test to what happens with an empty struct as well.

File: sdk/sdktypes/value_wrap_test.go (hunk 1 above)
Comment (ig-ra) [reply sdk/sdktypes/value_wrap_test.go]: added `struct{}{}` test
